### DISCUSSION TITLE
build failed

### DISCUSSION THREAD

**Discussion by MaliaCataleya** [eclipse-edc/Samples#23] [Q&A] [answered]
Hey!

When trying to build the repo right after I cloned it, an error comes up.

![image](https://user-images.githubusercontent.com/81524401/219329932-8e6c55cf-2c2a-452e-9e06-dd9e944ff078.png)

Can anyone help me out?

Steps so far: 
- installed JDK 11
- cloned repo
- ran './gradlew clean basic:basic-01-basic-connector:build' from Samples

**Comment by MaliaCataleya** [comment, marked as answer]
nvmd, fixed it :D

My JAVA_HOME system variable was pointing to java 8 while I referenced to the jdk 11 on the PATH variable. FYI ^^

**Reply by SebMarsolle** [reply]
I had similar problem,
I found that property javaVersion=11 in file gradle.properties must be aligned with the JDK version you use to run gradlew command.
You can even upgrade this version to javaVersion=17 if you have locally a JDK 17.
